ComCompany/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+namespace Company
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Driver manoliiDriver = new Driver("Манолiй", 6);
+            Driver janineDriver = new Driver("Яна", 1);
+            Engine tigershark_2_4 = new Engine
+            {
+                Power = 186,
+                Manufacturer = "Chrysler"
+            };
+            Engine mzr_2_0 = new Engine
+            {
+                Power = 147,
+                Manufacturer = "Mazda"
+            };
+            Engine diesel_2_0 = new Engine
+            {
+                Power = 140,
+                Manufacturer = "MAN"
+            };
+            Engine gasoline_1_5 = new Engine
+            {
+                Power = 150,
+                Manufacturer = "JAC"
+            };
+            Car jeepRenegade = new Car
+            {
+                Model = "Джип",
+                Class = "4х4",
+                Weight = 1400,
+                Driver = manoliiDriver,
+                Engine = tigershark_2_4
+            };
+            Car mazda6 = new Car
+            {
+                Model = "Мазда 6",
+                Class = "Клас D",
+                Weight = 1600,
+                Driver = janineDriver,
+                Engine = mzr_2_0
+            };
+            Lorry man_tge_3_140 = new Lorry
+            {
+                Model = "MAN TGE 3.140",
+                Class = "Фургон",
+                Weight = 3500,
+                Driver = manoliiDriver,
+                Engine = diesel_2_0,
+                CarryingCapacity = 1411
+            };
+            SportCar jac_j7 = new SportCar
+            {
+                Model = "JAC J7",
+                Class = "Лiфтбек D-класу",
+                Weight = 1500,
+                Driver = janineDriver,
+                Engine = gasoline_1_5,
+                MaxSpeed = 170
+            };
+
+            Console.WriteLine(jeepRenegade.Start());
+            Console.WriteLine(jeepRenegade.TurnRight());
+            Console.WriteLine(jeepRenegade.Stop());
+            Console.WriteLine(jeepRenegade.TurnLeft());
+            Console.WriteLine(jeepRenegade.ToString());
```

Comment:
try this - `Console.WriteLine(jeepRenegade);`

Reply:
it works, but I use methods in different order

Reply:
Exactly, so there is no need to call `ToString()` explicitly